### PR TITLE
AR-769: Poster

### PR DESCRIPTION
**Pull request by tuj**
#### Link to ticket

https://jira.itkdev.dk/browse/AR-769

#### Description

Added eventdatabasen feed source.

#### Checklist

- [ ] My code is covered by test cases.
- [x] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [x] My code passes our continuous integration process.

### CODE REVIEW

File: src/Feed/EventdatabasenFeedType.php [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+use App\Entity\Feed;
+use App\Entity\FeedSource;
+use App\Service\FeedService;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * See https://github.com/itk-event-database/event-database-api.
+ */
+class EventdatabasenFeedType implements FeedTypeInterface
```
Comment (turegjorup) [inline src/Feed/EventdatabasenFeedType.php:14]: "Eventdatabasen" should properply be shange to "DetSkerIAarhus", which is the official name: https://api.detskeriaarhus.dk/

*  not a mix between danish and english
* scopes it to Aarhus, which we should do when putting Aarhus specific things in "core"

File: src/Feed/EventdatabasenFeedType.php [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+        $configuration = $this->feedService->getFeedConfiguration($feed);
+
+        if (!isset($secrets['host'])) {
+            return [];
+        }
+
+        $host = $secrets['host'];
+
+        if (isset($configuration['posterType'])) {
+            if ('subscription' == $configuration['posterType']) {
```
Comment (turegjorup) [inline src/Feed/EventdatabasenFeedType.php:35]: Would prefer a `switch`

**Review by turegjorup — CHANGES_REQUESTED** [review]
Please rename to `DetSkerIAarhusFeedType`

File: src/Feed/EventdatabasenFeedType.php (hunk 1 above)
Comment (tuj) [reply src/Feed/EventdatabasenFeedType.php:14]: Changed to EventDatabaseApiFeedType

File: src/Feed/EventdatabasenFeedType.php (hunk 2 above)
Comment (tuj) [reply src/Feed/EventdatabasenFeedType.php]: done

**Review by turegjorup — APPROVED** [review]

**Pull request merged**